user-manager/src/main/java/com/qpay/usermanager/service/impl/CustomerServiceImpl.java

```diff
@@ -0,0 +1,56 @@
+package com.qpay.usermanager.service.impl;
+
+import com.qpay.usermanager.mapper.CustomerMapper;
+import com.qpay.usermanager.model.dto.customer.CustomerModification;
+import com.qpay.usermanager.model.entity.customer.CustomerEntity;
+import com.qpay.usermanager.repository.CustomerRepository;
+import com.qpay.usermanager.service.CustomerService;
+import com.qpay.usermanager.service.exception.CustomerAlreadyExistsException;
+import com.qpay.usermanager.service.exception.CustomerNotFoundException;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class CustomerServiceImpl implements CustomerService {
+
+    private final CustomerRepository customerRepository;
+
+    private final CustomerMapper customerMapper;
+
+
+    private boolean isExistCustomerByEmail(final String email) {
```

Comment:
name of the method is not correctly in english. 
```suggestion
    private boolean existsCustomerByEmail(final String email) {
```